src/styles/saas-list.less

```diff
@@ -1,76 +1,104 @@
+.build-applications-view {
+
+  h1 {
+    color: @saas-text;
+    margin: 0;
+    padding: 20px;
```

Comment:
Hi @rhamilto, just fyi that I initially created this .less file which is why it probably needs some cleaning up :-)  @cshinn is not to blame!!! :-)